app/lib/project/MetapolatorProject.js

```diff
@@ -614,6 +616,38 @@ define([
         exportController.export();
     };
 
+    _p.exportZipInstance = function(masterName, instanceName, precision) {
+        //export the font to a temporary directory
+        var temp_dir = instanceName+"_temp";
+        this.exportInstance(masterName, temp_dir, precision);
+
+        //encode it as a zip file
+        var content = zip.encode(this._io, temp_dir);
+        this._io.writeFile(false, instanceName, new Buffer(content, 'base64'));
+
+        //delete the temp dir
+        function rmDirRecursive(io, dir){
```

Comment:
On 13.04.2015 23:39, Felipe Corrêa da Silva Sanches wrote:
>
> In app/lib/project/MetapolatorProject.js
> <https://github.com/metapolator/metapolator/pull/396#discussion_r28282085>:
>
> > @@ -614,6 +616,38 @@ define([
> >          exportController.export();
> >      };
> >  
> > +    _p.exportZipInstance = function(masterName, instanceName, precision) {
> > +        //export the font to a temporary directory
> > +        var temp_dir = instanceName+"_temp";
> > +        this.exportInstance(masterName, temp_dir, precision);
> > +
> > +        //encode it as a zip file
> > +        var content = zip.encode(this._io, temp_dir);
> > +        this._io.writeFile(false, instanceName, new Buffer(content, 'base64'));
> > +
> > +        //delete the temp dir
> > +        function rmDirRecursive(io, dir){
>
> I implemented rmDirRecursive myself. But it was clearly influenced by
> io/readDirRecursive.js
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/metapolator/metapolator/pull/396/files#r28282085>.
>

@rrthomas added this to io/_base.js
https://github.com/graphicore/ufoJS/blob/master/lib/tools/io/_base.js#L214
I guess I was reminded that there was already something. So, knowing
that, you could have used here

```
this._io.rmDirRecursive(false, path).
```